src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml

```diff
@@ -185,13 +186,54 @@
       Grid.Row="1"
       x:Name="_projectList"
       Margin="0,8,0,0"
-      ScrollViewer.VerticalScrollBarVisibility="Visible"
+      ScrollViewer.VerticalScrollBarVisibility="Auto"
       PreviewKeyUp="ProjectList_PreviewKeyUp"
       ItemsSource="{Binding Projects}"
       Background="{DynamicResource {x:Static local:Brushes.ListPaneBackground}}"
       Foreground="{DynamicResource {x:Static local:Brushes.UIText}}"
       KeyboardNavigation.TabNavigation="Local"
       SelectionMode="Single">
+
+      <ListView.Style>
```

Comment:
can you checkout the style template from blend and see if it uses the aero theme too?